.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "archspec/json"]
 	path = archspec/json
-	url = https://github.com/archspec/archspec-json.git
+	url = https://github.com/paolotricerri/archspec-json.git
```

Comment:
This, of course, needs to be undone before merging :slightly_smiling_face:

Reply:
Of course :). Was just trying to make sure tests were running and passing. Happy to restore correct value as soon as the review about `cpuid` vs `cpupart` is closed.